webcompat/templates/about.html

```diff
@@ -2,27 +2,26 @@
 {% block title %}About{% endblock %}
 {% block body %}
 {% include "shared/nav.html" %}
-  <main class="wc-UIContent" role="main">
-    <section class="wc-UISection wc-UISection">
-      <h1 class="wc-Title wc-Title--l">About</h1>
-      <h2 class="wc-Title wc-Title--m">Webcompat.com is developed by volunteers and supported by Mozilla. This site is an <span class="wc-bold" style="color:grey">open invitation for all web users, developers and browser vendors</span> to get involved in the web compatibility effort. Our goal is to make it easy to report and view problems for any part of the web.</h2>
-      <div class="wc-Box-media">
-        <img class="u-image" src="../img/about.png" alt="About logo" />
+  <section class="grid">
+    <article class="grid-row">
+      <div class="grid-cell x3">
+        <h1 class="headline-1">About</h1>
+        <h2 class="headline-2">Webcompat.com is developed by volunteers and supported by Mozilla. This site is an <span class="bold">open invitation for all web users, developers and browser vendors</span> to get involved in the web compatibility effort. Our goal is to make it easy to report and view problems for any part of the web.</h2>
+        <h3 class="headline-3">What is Web Compatibility?</h3>
+        <p>Web compatibility is about making sure web sites work consistently across all browsers and devices.</p>
+        <p>Sometimes, sites have bugs or policies that prevent them from working well in every browser. We work to help web developers and site owners identify and fix such issues.</p>
+        <h3 class="headline-3">Web compatibility issues usually follow a three part life cycle:</h3>
+        <div>
```

Comment:
Are these divs needed? It looks kinda cramped in the frontend. Maybe replace them withi `<p>`s so we get a bit of spacing between the paragraphs?